src/index.ts

```diff
@@ -5,8 +5,9 @@ import './assets/images/search_button.png';
 import './assets/images/logo.png';
 import './assets/images/star_empty.png';
 import './assets/images/star_filled.png';
+import './assets/images/image.png';
 
 import App from './components/App/App';
 
-const app = new App();
+const app = App;
 document.body.appendChild(app.$target);
```

Comment:
할당하지 않고 바로 사용하면 어떨까용?

```suggestion
document.body.appendChild(App.$target);
```